src/Installer.php

```diff
@@ -262,6 +257,7 @@ public function createAssetSymlinks() {
       $finder->name($name);
     }
     $finder->exclude('sites/default/files');
+    $finder->notName('/\.php/');
```

Comment:
FYI I've extended the list to:

```+    $finder->notName('*.inc');
+    $finder->notName('*.install');
+    $finder->notName('*.module');
+    $finder->notName('*.php');
+    $finder->notName('*.profile');
+    $finder->notName('*.theme');```

https://github.com/drupal-composer/drupal-paranoia/commit/160a9337b91224eecf3e9671480a3cb1381e457c#diff-c984225cadcacb6a40cb298c78b32434R260